src/main/telemetry/hott.c

```diff
@@ -453,6 +530,565 @@ static void processBinaryModeRequest(uint8_t address)
 #endif
 }
 
+static void processTextModeRequest(HOTT_TEXTMODE_MSG_t *hottTEXTMODEMessage,uint8_t octet3)
+{
+	uint8_t id_sensor = (octet3 >> 4);
+	uint8_t id_key = octet3 & 0x0f;
+	static uint8_t ligne_select = 4 ;
+	static int8_t ligne_edit = -1 ;
+	char lineText[21];
+
+	hottTEXTMODEMessage->start_byte = 0x7b;
+	hottTEXTMODEMessage->esc = 0x00;
+	hottTEXTMODEMessage->warning_beeps = 0x00;
+	memset((char *)&hottTEXTMODEMessage->text, 0x20, HOTT_TEXTMODE_MSG_TEXT_LEN);
+	hottTEXTMODEMessage->stop_byte = 0x7D;
+	
+	if (id_key == HOTTV4_BUTTON_LEFT && page_settings == 1)
+        {   
```

Comment:
Please use K&R style - opening braces on a new line only for function definitons.